docs/source/tutorials/datasets.rst

```diff
@@ -0,0 +1,457 @@
+        "data_rows": [
+            {
+                "data_hash": "<data_hash>",
+                "data_title": "sample_data.mp4",
+                "data_type": "VIDEO"
+            }, {
+                "data_hash": "<data_hash>",
+                "data_title": "image-group-abcde",
+                "data_type": "IMG_GROUP"
+            },
+            ...
+        ]
+    }
+
+Note how the ``data_rows`` can both consist of videos and image groups.
+
+Below, you can find tutorials on how to interact with your datasets.
+
+
+Creating a dataset
+==================
+
+You can use the |sdk| to create a dataset.
+First, you need to create a :xref:`public-private_key_pair` for |company|.
+
+You also need to select where your data will be hosted to select the appropriate :class:`StorageLocation <encord.orm.dataset.StorageLocation>`.
+For example, the following example will create a dataset called "traffic data" that will expect data hosted on AWS S3.
+
+.. tabs::
+
+    .. tab:: Code
+
+        .. literalinclude:: /code_examples/tutorials/datasets/creating_a_dataset.py
+            :language: python
+
+    .. tab:: Example output
+
+        .. code-block:: python
+
+            {
+                "title": "Example Title",
+                "type": 1,
+                "dataset_hash": "<dataset_hash>",
+                "user_hash": "<user_hash>",
+            }
+
+
+Listing existing datasets
+=========================
+
+Via the :class:`.EncordUserClient`, you can easily query and list all the available datasets of a given user.
+In the example below, a user authenticates with |company| and then fetches all datasets available.
+
+
+.. autolink-concat:: section
+
+.. autolink-preface::
+    from encord.orm.dataset import DatasetInfo, DatasetUserRole, StorageLocation
+
+
+.. tabs::
+
+    .. tab:: Code
+
+        .. literalinclude:: /code_examples/tutorials/datasets/listing_existing_datasets.py
+            :language: python
+
+    .. tab:: Example output
+
+        .. code-block:: python
+
+            [
+                {
+                    "dataset": DatasetInfo(
+                            dataset_hash="<dataset_hash>",
+                            user_hash="<user_hash>",
+                            title="Example title",
+                            description="Example description ... ",
+                            type=0,  # encord.orm.dataset.StorageLocation
+                            created_at=datetime.datetime(...),
+                            last_edited_at=datetime.datetime(...)
+                        ),
+                    "user_role": DatasetUserRole.ADMIN
+                },
+                # ...
+            ]
+
+*Note:* the ``type`` attribute in the output refers to the :class:`.StorageLocation` used when :ref:`tutorials/datasets:Creating a dataset`.
+
+.. note::
+
+    :meth:`.EncordUserClient.get_datasets` has multiple optional arguments that allow you to query datasets with specific characteristics.
```

Comment:
Might be worth saying something like 'Please refer to the SDK reference' for more information, or something to that effect.

Reply:
The :meth:... is effectively a link to exactly where they would want to look.

Reply:
Ah ok